v2/sourcedb-to-spanner/src/main/java/com/google/cloud/teleport/v2/source/reader/io/jdbc/iowrapper/JdbcIoWrapper.java

```diff
@@ -220,6 +221,21 @@ private static ImmutableList<TableConfig> autoInferTableConfigs(
     return tableConfigsBuilder.build();
   }
 
+  static ImmutableList<String> getTablesToMigrate(
```

Comment:
[nit] would it be possible to make `getTablesToMigrate` private (and Visible for testing) or protected?
As of now this is called only from a private method within the same class.